django_ses_plus/__init__.py

```diff
@@ -1,5 +1,3 @@
 import logging
 
 logger = logging.getLogger('django_ses_plus')
-
-default_app_config = 'django_ses_plus.apps.DjangoSESPlusConfig'
```

Comment:
Deprecated in Django 3.2, https://docs.djangoproject.com/en/4.1/releases/3.2/#automatic-appconfig-discovery